code/game/objects/structures/crates_lockers/closets/secure/freezer.dm

```diff
@@ -3,6 +3,27 @@
 	var/icon_exploded = "fridge_exploded"
 	var/exploded = 0
 
+/obj/structure/closet/secure_closet/freezer/New()
+	..()
+	processing_objects.Add(src)
+
```

Comment:
Need a `Destroy()` to remove from processing objects too.

Reply:
obj/Destroy does it, though.

Reply:
I did not know this.

Reply:
So, disregard this change request?